templates/CMakeLists.txt.template

```diff
@@ -68,6 +68,17 @@
       return ''
     return 'endif()\n'
   %>
+  <%
```

Comment:
nit: probably deserves a comment?

Reply:
Not a bad idea. I'll add it once the tests finish.

Reply:
Makes sense. I actually meant a comment that explains that we prepare the set of all the .proto files to generate to avoid using protobuf_generate_grpc_cpp() multiple times for a single .proto, which is something that's illegal in cmake. Can we still add that one?